tests/DatasetsTests/smokeTest.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { expect } from "@playwright/test";
+import { test } from "../../fixture/fixture"; 
+
+test('Open Main Page', async ({datasetsPage}) => {
+    await datasetsPage.openMainPage();
+});
+test('Open Datasets Page', async ({datasetsPage}) => {
+    await datasetsPage.openDatasetsPage();
```

Comment:
1) Please add some asserts here
2) There are also no download actions (based on the task of your group)